vsintegration/src/FSharp.Editor/Common/LanguageService.fs

```diff
@@ -48,7 +48,7 @@ type internal FSharpCheckerProvider
     ) =
     let checker = 
         lazy
-            let checker = FSharpChecker.Create()
+            let checker = FSharpChecker.Create(projectCacheSize = 200, keepAllBackgroundResolutions = false)
```

Comment:
Also, I would very much like to see this setting be user-settable, or at least controlled by an environment variable or something.  Given the major features added in VS2017, I feel it's inevitable that we will get reports that the F# support now uses more memory (and hence becomes unusable more often). We really need to be able to do on-machine diagnosis and treatment when that happens.